README.md

```diff
@@ -1 +1,103 @@
-# Connect-Four
\ No newline at end of file
+# Connect-Four
+
+## Table of contents
+
+- [Task List](#Task-List)
+- [BoardState (Class)](#BoardState-(Class))
+  - [board](#board-(Field))
+  - [valid_moves](#valid_moves-(Accessor))
+  - [game_status](#game_status-(Accessor))
+  - [current_turn](#current_turn-(Accessor))
+  - [display_board](#display_board-(Accessor))
+  - [Constructor](#Constructor)
+  - [update](#update-(Mutator))
+- [Player (Class)](#Player-(Class))
+  - [player_type](#player_type-(Field))
+  - [make_move](#move_process-(Field))
+- [Game (Class)](#Player-(Class))
+  - [board_state](#board_state-(Field))
+  - [player_one](#player_one-(Field))
+  - [player_two](#player_two-(Field))
+  - [game_history](#game_history-(Field))
+  - [do_display](#do_display-(Field))
+  - [replay](#replay-(Accessor))
+  
+## Operations Concept
+- Start Game
+- Choose Player One 
+- Choose Player Two
+- Choose Visualization
+- Player 1 moves
+- Update Board
+- Check Game Status
+- If Continues, Player 2 moves
+- If Ended (Tie or Winner), Exit Interface
+- 
+- Exit Interface
+- Write Game History to File?
+- Replay the Game History
+- Play Again?
+
+
+## Task List
```

Comment:
Task List is Incomplete and need an assignee for each task

Reply:
Cool, we will update